resources/app.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
 dependencies:
```

Comment:
What I think about version dependencies:
we should add strong dependency here and delete from anypoint application